test-cloud/pom.xml

```diff
@@ -124,6 +136,14 @@
               <default.domain.suffix>${default.domain.suffix}</default.domain.suffix>
               <kie.artifact.version>${kie.artifact.version}</kie.artifact.version>
               <kie.app.template.url>${kie.app.template.url}</kie.app.template.url>
+              <sso.image.streams>${sso.image.streams}</sso.image.streams>
+              <sso.app.secrets>${sso.app.secrets}</sso.app.secrets>
+              <sso.app.template>${sso.app.template}</sso.app.template>
+              <sso.app.realm>${sso.app.realm}</sso.app.realm>
+              <sso.admin.username>${sso.admin.username}</sso.admin.username>
+              <sso.admin.password>${sso.admin.password}</sso.admin.password>
+              <sso.service.username>${sso.service.username}</sso.service.username>
+              <sso.service.password>${sso.service.password}</sso.service.password>
```

Comment:
It seems that org.kie.workbench.maven.* parameters aren't passed to failsafe

Reply:
good catch! Workbench maven user parameters added to failsafe.